resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -293,10 +293,14 @@
 <?php endif ?>
 
 @if ($gate)
+    <?php
+        $matureHubIcon = Game::where('Title', '[Theme - Mature]')->get(['ImageIcon']);
```

Comment:
This is overfetching for the entire model and then narrowing it to a single field. `value()` will perform better.

Reply:
Change applied in latest.